packages/next-swc/crates/next-core/src/next_shared/resolve.rs

```diff
@@ -16,9 +17,27 @@ use turbopack_binding::{
     },
 };
 
+use crate::next_telemetry::ModuleFeatureTelemetry;
+
 lazy_static! {
     static ref UNSUPPORTED_PACKAGES: HashSet<&'static str> = ["@vercel/og"].into();
     static ref UNSUPPORTED_PACKAGE_PATHS: HashSet<(&'static str, &'static str)> = [].into();
+    // Set of the features we want to track, following existing references in webpack/plugins/telemetry-plugin.
+    static ref FEATURE_MODULES: HashMap<&'static str, Vec<&'static str>> = HashMap::from([
+        (
+            "next",
+            vec![
+                "/image",
+                "future/image",
+                "legacy/image",
```

Comment:
Why do these items not have a leading slash?

Reply:
oh, looks like a mistake.